tutorials/nextjs-app-serverless-functions-sqldb/index.mdx

```diff
@@ -0,0 +1,948 @@
+---
+meta:
+  title: Deploy a Next.js application using Serverless Containers and Serverless SQL Databases
+  description: This page explains how to deploy your fully serverless Next.js application using Serverless Containers and Serverless SQL Databases
```

Comment:
```suggestion
  description: This page explains how to deploy your serverless Next.js application entirely, using Serverless Containers and Serverless SQL Databases
```

Reply:
I'm not sure I'm familiar with what "fully" means in this context, feel free to ignore if my suggestion is irrelevant